static/bidder-info/smarthub.yaml

```diff
@@ -1,6 +1,6 @@
-endpoint: "http://prebid.smart-hub.io/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"
+endpoint: "https://prebid.attekmi.com/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"
 maintainer:
-  email: "support@smart-hub.io"
+  email: "prebid@attekmi.com"
```

Comment:
We sent a test email to this address. Please respond back with "Confirmed".

Reply:
Email was confirmed